packages/editor/src/plugins/rows/utils/static-is-empty.ts

```diff
@@ -1,10 +1,33 @@
 // eslint-disable-next-line import/no-cycle
-import type { AnyEditorDocument } from '@editor/types/editor-plugins'
+import type {
+  AnyEditorDocument,
+  EditorArticleDocument,
+  EditorArticleIntroductionDocument,
+} from '@editor/types/editor-plugins'
 import { isRowsDocument } from '@editor/types/plugin-type-guards'
 
 import { isEmptyTextDocument } from '../../text/utils/static-is-empty'
 
-export function isEmptyRowsDocument(rows: AnyEditorDocument) {
+export function isEmptyArticle(article: EditorArticleDocument): boolean {
```

Comment:
Can you move this function into a separate util file or into the article plugin? This file is within the rows-plugin.